rpc/client_example_test.go

```diff
@@ -31,11 +32,27 @@ import (
 // eth_getBlockByNumber("latest", {})
 //    returns the latest block object.
 //
-// eth_subscribe("newBlocks")
+// eth_subscribe("newHeads")
 //    creates a subscription which fires block objects when new blocks arrive.
 
 type Block struct {
-	Number *big.Int
+	Number *BigInt
+}
+
+type BigInt struct {
```

Comment:
I think it's confusing to call this `BigInt`. Let's call it something like `BlockNumberReply`, which makes it more intuitive that it may be either an actual `big.Int` or an error message. 
Unless there's some more standardized way to catch the RPC error, feels like there should be... ? cc @fjl ?